eng/scripts/packagevalidation/PrePublish-Validation.ps1

```diff
@@ -0,0 +1,97 @@
+[CmdletBinding()]
+Param (
+  [ValidateNotNullOrEmpty()]
+  [string] $PackagePath,
+  [string] $ArtifactName,
+  [string] $WorkingDirectory
+)
+
+. "${PSScriptRoot}\..\common\scripts\logging.ps1"
+. "${PSScriptRoot}\..\common\scripts\SemVer.ps1"
+
+$WinVersion = (Get-ChildItem -Path "C:\Program Files (x86)\Windows Kits\10\bin\10.0*" | Sort-Object -Descending)[0]
+$SignTool = "C:\Program Files (x86)\Windows Kits\10\bin\$($WinVersion.Name)\x64\signtool.exe"
+
+New-Item -Path $WorkingDirectory -Name "Validation" -ItemType "directory"
+$ValidationDirectory = "$WorkingDirectory\Validation"
+
+Copy-Item -Path $PackagePath -Destination "$ValidationDirectory\package.zip"
```

Comment:
Why copy the zip as opposed to just extracting it from where it is?

Reply:
Its initially a .nuget file. The Copy also transforms it to a zip.

Reply:
I suspect you can expand it as a nuget file as well I don't think the Expand-Archive will fail on files that don't end in zip.

Reply:
You are right. We don't need the copy step.